sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/Directory.Build.props

```diff
@@ -0,0 +1,25 @@
+﻿<Project>
```

Comment:
I think we should standardize this to match other libraries, e.g. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Directory.Build.props